### DISCUSSION TITLE
anyone have insight into the spotify analysis engine?

### DISCUSSION THREAD

**Discussion by jamesjsewell** [MTG/essentia#1093] [Q&A] [answered]
i'm wondering how essentia compares to the spotify engine or how spotify is analyzing their tracks?

**Comment by dbogdanov** [comment, marked as answer]
Hi @jamesjsewell, 

The difference is in different feature sets and the fact that Spotify API contains closed-sourced algorithms, so we can not know their implementation, nor can we customize those.

Essentia is more flexible in that respect as you can tune it for your tasks. Also, it provides a diverse set of features and algorithms for use-cases not covered by Spotify API.

If you have questions about particular descriptors, we can consult you further.
